components/style/matching.rs

```diff
@@ -288,17 +291,43 @@ trait PrivateMatchMethods: TElement {
         }
     }
 
+    /// Get the ComputedValues (if any) for our inheritance parent.
+    fn get_inherited_values_and_parent(&self) -> ParentElementAndValues<Self> {
```

Comment:
Even though the type is `ComputedValues`, we tend to use the name `style` when talking about it in the code currently, so I'd say call this `get_inherited_style_and_parent` and the type `ParentElementAndStyle` with a `style` field for the CV type.